packages/filebrowser/src/listing.ts

```diff
@@ -514,6 +514,21 @@ export class DirListing extends Widget {
       });
   }
 
+  /**
+   * Select all directory listing items
```

Comment:
Is this?

```suggestion
   * Select all listing items in the current directory.
```

I wonder, how it would behave with `jupyterlab-unfold` installed.